### PR TITLE
fix(gzip): change gzip=force url parameter to force_gzip=true

### PR DESCRIPTION
**Pull request by janl**

### CODE REVIEW

**Comment by gr2m** [comment]
do we still need `lib/assets/force-gzip.html`? I think we can remove it, now that the frontend code won't use the iframe hack, right?

File: test/integration/force-gzip.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ describe('handle forced gzip', function () {
       host: '127.0.0.1',
       port: config.www_port,
       method: 'get',
-      path: '/_api/?gzip=force',
+      path: '/_api/?force_gzip=true',
```
Comment (gr2m) [inline test/integration/force-gzip.js]: Would it make sense to add this here?

```
headers: {
  'Accept-Encoding': 'deflate'
}
```

Just to make clear that the request did not request gzip in its headers?

**Comment by gr2m** [comment]
see comments, but looks good :+1:

File: test/integration/force-gzip.js (hunk 1 above)
Comment (janl) [inline test/integration/force-gzip.js]: nah, this is alright. :)

**Comment by janl** [comment]
good point about the html file, and the cookie test in api/index.js, since we don’t use cookies for this now.

**Comment by janl** [comment]
last push includes removal of force-gzip.html and cookie handling

**Pull request merged**